index.html

```diff
@@ -69,6 +69,39 @@
       </div>
     </div>
 
+    <!-- APD community service sites filters -->
+    <div class="row">
+      <div class="col-xs-12">
+        <p><strong>Filter for APD community service sites.</strong> You cannot filter by age, language, fcaility, programs, or payment preferences.</p>
+        <div class="checkbox">
+          <label>
+            <p><input id="communitySite" type="checkbox">Show only APD community service sites</p>
+          </label>
+        </div>
+      </div>
+    </div>
+    <!-- Panel made visible after clicking #communitySite -->
+    <div class="row apd-filters hidden">
+      <div class="col-xs-12">
+        <div class="row">
+          <div class="col-md-4">
+            <div class="checkbox">
+              <label>
+                <p><input id="handicapAccessible" type="checkbox">Handicap accessible</p>
+              </label>
+            </div>
+          </div>
+          <div class="col-md-8">
+            <div class="checkbox">
+              <label>
+                <p><input id="acceptsSexOffenders" type="checkbox">Accepts sex offenders</p>
+              </label>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
     <div>
       <button class='btn btn-search' id="btnSearch"><i class='fa fa-search'></i> Search</button>
       <a href="/" class='btn btn-reset' id="btnReset" data-toggle="tooltip" data-placement="bottom" title="Reset"><i class="fa fa-repeat" aria-hidden="true"></i></a>
```

Comment:
These tooltips are cut off. The footer size should be increased.
<img width="426" alt="screen shot 2019-03-06 at 1 38 29 pm" src="https://user-images.githubusercontent.com/6961258/53908808-cbd90880-4015-11e9-98e5-41c4a73f640e.png">